src/client/Microsoft.Identity.Client/Cache/ITokenCacheAccessor.cs

```diff
@@ -42,7 +43,7 @@ internal interface ITokenCacheAccessor
         /// It should only support external token caching, in the hope that the external token cache is partitioned.
         /// Not all classes that implement this method are required to filter by partition (e.g. mobile)
         /// </remarks>
-        IReadOnlyList<MsalAccessTokenCacheItem> GetAllAccessTokens(string optionalPartitionKey = null);
+        IReadOnlyList<MsalAccessTokenCacheItem> GetAllAccessTokens(string optionalPartitionKey = null, ICoreLogger requestlogger = null);
```

Comment:
@trwalke - I propose that we have separate PRs, one for the improvements that you have made by using the requestLogger everywhere. This can easily be merged into MSAL at any time.

Reply:
ok, will do